### PR TITLE
fix sourcemaps so they refer to the original source

### PR DESCRIPTION
**Pull request by ehartford**
Make it possible to debug with the original source files rather than the concatted file.  

Necessary to update grunt-contrib-concat for this feature.

    npm install

### CODE REVIEW

File: src/models/forceDirectedGraph.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ nv.models.forceDirectedGraph = function() {
         , theta = 0.8
         , alpha = 0.1
         , radius = 5
+        , getX = d3.functor(0.0)
+        , getY = d3.functor(0.0)
```
Comment (liquidpele) [inline src/models/forceDirectedGraph.js]: This is unrelated, please keep PRs to particular features.

Comment (ehartford) [inline src/models/forceDirectedGraph.js]: ok

**Comment by liquidpele** [comment]
I'm getting this error after this change:

```Running "uglify:js" (uglify) task
{ [Error: Unable to read "build/nv.d3.js.map" file (Error code: ENOENT).]
  origError: 
   { [Error: ENOENT, no such file or directory 'build/nv.d3.js.map']
     errno: 34,
     code: 'ENOENT',
     path: 'build/nv.d3.js.map',
     syscall: 'open' } }
>> Uglifying source build/nv.d3.js failed.
Warning: Uglification failed.
Unable to read "build/nv.d3.js.map" file (Error code: ENOENT).. 
 Use --force to continue.

Aborted due to warnings.
```

Please fix or I'll have to revert your PR.

**Comment by ehartford** [comment]
you need to do "npm install" to get the updated grunt-contrib-concat

**Comment by liquidpele** [comment]
Haha in my defense I haven't had my coffee yet
